spice/cfg/tests-variants.cfg

```diff
@@ -471,7 +471,7 @@ variants:
             script = screenshot.rv --mouse
             include join.cfg
 
-        - 326060:
+        - 326060:    # not working for rhel6 client #BZ1351890
```

Comment:
you can put filter like:
- 326060:
no RHEL.6 # not working for rhel6 client #BZ1351890